src/System.Text.RegularExpressions/src/Resources/Strings.resx

```diff
@@ -88,6 +88,9 @@
   <data name="IllegalCondition" xml:space="preserve">
     <value>Illegal conditional (?(...)) expression.</value>
   </data>
+  <data name="IllegalDefaultRegexMatchTimeoutInAppDomain" xml:space="preserve">
+    <value>AppDomain data '{0}' contains an invalid value or object for specifying a default matching timeout for System.Text.RegularExpressions.Regex.</value>
```

Comment:
maybe include the value as well as the name?

Reply:
will do.